ngraph/frontend/paddlepaddle/src/op/deformable_conv.cpp

```diff
@@ -0,0 +1,76 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "deformable_conv.hpp"
+#include <ngraph/opsets/opset8.hpp>
+#include "conv2d_utils.hpp"
+
+namespace ngraph
+{
+    namespace frontend
+    {
+        namespace pdpd
+        {
+            namespace op
+            {
+                NamedOutputs deformable_conv(const NodeContext& node)
+                {
+                    auto input = node.get_ng_input("Input");
+                    auto filter = node.get_ng_input("Filter");
+                    auto offset = node.get_ng_input("Offset");
+
+                    auto strides = node.get_attribute<std::vector<int>>("strides");
+                    auto dilations = node.get_attribute<std::vector<int>>("dilations");
+
+                    auto groups = node.get_attribute<int>("groups");
+                    auto deformable_groups = node.get_attribute<int>("deformable_groups");
+                    // auto im2col_step = node.get_attribute<int>("im2col_step"); // TODO
```

Comment:
What is the meaning of this line? Could you provide more description.

Reply:
Thanks for reminder. Confirmed, useless comment line. Removed.